pkg/elastic/client.go

```diff
@@ -96,6 +96,17 @@ func (c *Client) ListDeploymentRoles(ctx context.Context) (map[string]Deployment
 	return res, nil
 }
 
+// ListDeploymentRoleMapping returns a list of all Elastic roles on deployment.
+func (c *Client) ListDeploymentRoleMapping(ctx context.Context) (map[string]DeploymentRole, error) {
+	res := make(map[string]DeploymentRole)
+	usersUrl, _ := url.JoinPath(c.deploymentEndpoint, "_security/role_mapping")
```

Comment:
I think this variable should be named something else. It's not a url that returns users, right?

Reply:
Fixed